benchmarks/bench.sh

```diff
@@ -548,20 +544,19 @@ data_tpch() {
         echo "Internal error: Scale factor not specified"
         exit 1
     fi
+    FORMAT=$2
```

Comment:
```suggestion
    FORMAT=${2:-parquet}
```

Reply:
Ah, I see what you are saying now - there are several other calls to data_tpch that don't pass the format argument here

https://github.com/alamb/datafusion/blob/21a0237a1b96cf42bd96e93e6fc184a8e320138f/benchmarks/bench.sh#L298-L308

eg
```sh
                sort_tpch)
                    # same data as for tpch
                    data_tpch "1"
                    ;;
                sort_tpch10)
                    # same data as for tpch10
                    data_tpch "10"
                    ;;
                topk_tpch)
                    # same data as for tpch
                    data_tpch "1"
                    ;;
                nlj)
```


I will update those calls to explicitly pass in a format as I think that will make it clear what is going on